www/templates/account/includes/signup.php

```diff
@@ -329,6 +336,18 @@
 <script src="https://js.braintreegateway.com/web/dropin/1.33.0/js/dropin.min.js"></script>
 
 <script>
+const radios = document.querySelectorAll('input[name="plan"]')
```

Comment:
Do we need to add a listener for each radio button? Wondering if a single listener at a parent element might be easier?

maybe that's too much and we can worry about it later